tests/integration/aminojson/aminojson_test.go

```diff
@@ -294,8 +294,6 @@ var (
 
 		// upgrade
 		genType(&upgradetypes.Plan{}, &upgradeapi.Plan{}, genOpts.WithDisallowNil()),
-		genType(&upgradetypes.SoftwareUpgradeProposal{}, &upgradeapi.SoftwareUpgradeProposal{}, genOpts.WithDisallowNil()),
```

Comment:
should we keep this since its depreacted anyways?

Reply:
I think we should keep it yes.